### PR TITLE
Add more comments to file

### PR DESCRIPTION
**Pull request by kmoscoe**
Added note re GCS output dirs and moved NL model info from doc to here.

### CODE REVIEW

File: custom_dc/env.list.sample [hunk 1]
```diff
@@ -48,7 +49,8 @@ GOOGLE_ANALYTICS_TAG_ID=
 
 ### Variables that rarely need to be changed ###
 
-# Enable embeddings generation and natural language querying.
+# Enable embeddings generation and natural language querying. If you find that container startup
```
Comment (keyurva) [inline custom_dc/env.list.sample:52]: Let's not add the additional comment. They lose NL if they set it to false.

File: custom_dc/env.list.sample [hunk 2]
```diff
@@ -12,9 +12,10 @@ MAPS_API_KEY=
 # Local paths must be absolute paths.
 INPUT_DIR=
 # Output directory in Google Cloud Storage (GCS) or local drive for generated csv files.
-# GCS paths must be of the form gs://<BUCKET>/<FOLDER>.
+# GCS paths must be of the form gs://<BUCKET>/<FOLDER>. If you specify a GCS path, be 
+# sure to set USE_CLOUDSQL=true.
```
Comment (keyurva) [inline custom_dc/env.list.sample:16]: They need to set more than USE_CLOUDSQL. So how about: If you specify a GCS path, you need to use Cloud SQL.

File: custom_dc/env.list.sample [hunk 3]
```diff
@@ -12,9 +12,10 @@ MAPS_API_KEY=
 # Local paths must be absolute paths.
 INPUT_DIR=
 # Output directory in Google Cloud Storage (GCS) or local drive for generated csv files.
-# GCS paths must be of the form gs://<BUCKET>/<FOLDER>.
+# GCS paths must be of the form gs://<BUCKET>/<FOLDER>. If you specify a GCS path, be
```
Comment (keyurva) [inline custom_dc/env.list.sample:15]: nit: Perhaps start the additional sentence on a new line.

File: custom_dc/env.list.sample (hunk 1 above)
Comment (kmoscoe) [reply custom_dc/env.list.sample:52]: SG.

File: custom_dc/env.list.sample (hunk 3 above)
Comment (kmoscoe) [reply custom_dc/env.list.sample:15]: Will do.

File: custom_dc/env.list.sample (hunk 2 above)
Comment (kmoscoe) [reply custom_dc/env.list.sample:16]: Good point, I will actually point to relevant parameters section.

**Pull request closed without merging**